### PR TITLE
[CBRD-24396]modified answer for index hint.

### PR DESCRIPTION
**Pull request by tw-kang**
http://jira.cubrid.org/browse/CBRD-24396

### CODE REVIEW

File: sql/_13_issues/_12_2h/answers/bug_bts_9935_03.answer [hunk 1]
```diff
@@ -596,13 +596,13 @@ a    b    c
 Query plan:
 temp(group by)
     subplan: idx-join (inner join)
-                 outer: sscan
-                            class: t? node[?]
+                 outer: iscan
```
Comment (tw-kang) [inline sql/_13_issues/_12_2h/answers/bug_bts_9935_03.answer]: need revert

File: sql/_13_issues/_20_2h/answers/cbrd_21452.answer [hunk 2]
```diff
@@ -26,17 +26,16 @@ select test?.a from test? test? where (inst_num()<= ?:? )
 Query plan:
 sscan
     class: test? node[?]
-    sargs: term[?]
     cost:  ? card ?
 Query stmt:
-select test?.a from test? test? where inst_num()<=?
+select test?.a from test? test?
```
Comment (tw-kang) [inline sql/_13_issues/_20_2h/answers/cbrd_21452.answer]: need revert

File: sql/_24_aprium_qa/_03_other/issue_6091_orderbynum/answers/_06_mro_partition_orderbynum.answer [hunk 3]
```diff
@@ -44,12 +44,13 @@ i2    i3    i
 Query plan:
 temp(order by)
     subplan: idx-join (inner join)
-                 outer: sscan
-                            class: u node[?]
+                 outer: iscan
```
Comment (tw-kang) [inline sql/_24_aprium_qa/_03_other/issue_6091_orderbynum/answers/_06_mro_partition_orderbynum.answer]: need revert

File: sql/_24_aprium_qa/_03_other/issue_6091_orderbynum/answers/_07_mro_limit_orderbynum.answer [hunk 4]
```diff
@@ -44,12 +44,13 @@ i2    i3    i
 Query plan:
 temp(order by)
     subplan: idx-join (inner join)
-                 outer: sscan
-                            class: u node[?]
+                 outer: iscan
```
Comment (tw-kang) [inline sql/_24_aprium_qa/_03_other/issue_6091_orderbynum/answers/_07_mro_limit_orderbynum.answer]: need revert

File: sql/_24_aprium_qa/_03_other/issue_7441_MRO/answers/_01_mro_joins.answer [hunk 5]
```diff
@@ -250,30 +250,28 @@ Query stmt:
...
+1     1     2     1     1     1     2     
+1     1     2     1     1     1     1     
+3     1     2     1     1     3     2     
 
 Query plan:
 temp(order by)
-    subplan: nl-join (inner join)
-                 edge:  term[?]
-                 outer: iscan
+    subplan: idx-join (inner join)
```
Comment (tw-kang) [inline sql/_24_aprium_qa/_03_other/issue_7441_MRO/answers/_01_mro_joins.answer]: need revert

File: sql/_24_aprium_qa/_03_other/issue_7441_MRO/answers/_06_mro_partition.answer [hunk 6]
```diff
@@ -37,12 +37,13 @@ i2    i3    i
 Query plan:
 temp(order by)
     subplan: idx-join (inner join)
-                 outer: sscan
-                            class: u node[?]
+                 outer: iscan
```
Comment (tw-kang) [inline sql/_24_aprium_qa/_03_other/issue_7441_MRO/answers/_06_mro_partition.answer]: need revert

File: sql/_24_aprium_qa/_03_other/issue_7441_MRO/answers/_07_mro_limit.answer [hunk 7]
```diff
@@ -47,12 +47,13 @@ i2    i3    i
 Query plan:
 temp(order by)
     subplan: idx-join (inner join)
-                 outer: sscan
-                            class: u node[?]
+                 outer: iscan
```
Comment (tw-kang) [inline sql/_24_aprium_qa/_03_other/issue_7441_MRO/answers/_07_mro_limit.answer]: need revert

File: sql/_28_features_930/issue_13133_hash_query_profile/answers/hash_agg_query_profile.answer [hunk 8]
```diff
@@ -244,32 +244,59 @@ a    b
...
+    },
+    {
       "TABLE SCAN": {
         "table": "__t?"
-      }
-    },
-    "rewritten query": "select [__t?].a, [__t?].b, [__t?].c, [__t?].d from [dba.t?] [__t?] order by ?, ?, ?, ? for (orderby_num())<= ?:? "
-  },
+      },
+      "rewritten query": "select [__t?].a, [__t?].b from (select [dba.t?].a, [dba.t?].b, [dba.t?].c, [dba.t?].d from [dba.t?] [dba.t?] order by ?, ?, ?, ?) [__t?] (a, b, c, d) where (inst_num()<= ?:? )"
```
Comment (tw-kang) [inline sql/_28_features_930/issue_13133_hash_query_profile/answers/hash_agg_query_profile.answer]: need revert

File: sql/_33_elderberry/cbrd_24011/answers/inst_num.answer [hunk 9]
```diff
@@ -182,21 +182,20 @@ temp(group by)
...
-select tab_b.col_a from tab_b tab_b group by tab_b.col_a having groupby_num()<= ?:? 
+select tab_b.col_a from tab_b tab_b group by tab_b.col_a
 Query plan:
 sscan
     class: tab_b node[?]
-    sargs: term[?]
     cost:  ? card ?
 Query stmt:
-select tab_b.col_a from tab_b tab_b where inst_num()<=?
+select tab_b.col_a from tab_b tab_b
```
Comment (tw-kang) [inline sql/_33_elderberry/cbrd_24011/answers/inst_num.answer]: need revert

**Review by ctshim — APPROVED** [review]

**Pull request merged**